platform/windows/os_windows.cpp

```diff
@@ -1088,14 +1088,14 @@ void OS_Windows::initialize(const VideoMode& p_desired,int p_video_driver,int p_
 	RasterizerGLES3::make_current();
 #else
 	// FIXME: Does DX support still work now that rasterizer is no longer used?
- #ifdef DX9_ENABLED
-	rasterizer = memnew( RasterizerDX9(hWnd) );
- #endif
+	#ifdef DX9_ENABLED
```

Comment:
Yes, with `git commit --amend` to update the last commit (HEAD), or `git rebase -i HEAD~3` to rebase the last 3 commits interactively, which lets you define if you want to drop, edit or squash commits.